### DISCUSSION TITLE
Multiple Sample Violin Plot

### DISCUSSION THREAD

**Discussion by simonekats** [satijalab/seurat#5968] [Q&A]
Hello all,

I am trying to create a violin plot that displays multiple patients samples on the x axis, with them labeled, and just one feature at a a time. I have merged the data samples together, but the only identifier I have is "add.cells.ids". I have no idea how to access this and have it display in a VLN plot. Any ideas? Thanks!

**Comment by timoast** [comment]
You need something in the metadata that will group cells according to their sample of origin. Then you can use the `group.by` parameter in `VlnPlot()` to group the cells according to that variable. Metadata can be added to a Seurat object using the `AddMetaData()` function, or using the `$` assignment operator, for example `obj$dataset <- "sample_1"`
